docs/using-mina/how-to-send-and-receive.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: How to Send & Receive
+---
+
+import Subhead from "@site/src/components/common/Subhead";
+
+# How to Send & Receive
+
+<Subhead>
+Learn how to send and receive MINA
+</Subhead>
+
+
+### Prerequisites
+
+First, ensure you have [installed a Mina wallet](../using-mina/install-a-wallet).
+
+### Receiving MINA
+
+To receive MINA, you’ll need to locate the unique address for your Mina account and share this with the sender.
+
+Here’s how to find your address within common wallets:
+
+- [**Auro**](https://www.aurowallet.com) - Open the wallet, tap “*Receive*”, and tap “*Copy*” to share your address or QR code representing this address.
+- [**Clorio Wallet**](https://clor.io) - Open the wallet and tap <img src="/img/Copy_icon.png" alt="Copy icon on Clor.io" width="3%"/> at the top to copy your address.
+- [**Staking Power**](https://www.staking-power.com) - Open the wallet, tap “Receive”, and tap <img src="/img/Copy_icon_2.png" alt="Copy icon on Staking Power" width="3%"/> or “*Share*” to copy your address or QR code representing this address.
+- [**Ledger Hardware Wallet**](https://www.ledger.com/mina-wallet) - Follow the instructions on creating a Ledger Mina account via Auro Wallet [here](https://support.ledger.com/hc/en-us/articles/5458493215901-Creating-a-Ledger-MINA-account-via-Auro-Wallet?docs=true), then see the instructions for receiving MINA on Auro wallet above.
+
+<br></br>
+<center>
+<figure>
+  <img
+    src="/img/receive_MINA.png"
+    alt="Screenshot of Auro Wallet"
+    width="58%"
+  />
+</figure>
+</center>
+<br></br>
+
+:::note
+
+The Mina network charges a one-time account creation fee of 1 MINA for spam prevention. It will be automatically deducted from the first transaction received.
+
+:::
+
+### Sending MINA
+When sending MINA, always double check that you have the correct Mina address for the recipient.  Remember that <u>transactions on the Mina blockchain are final and irreversible</u>. 
```

Comment:
Would consider using italic instead of underline just to avoid people thinking it looks like a non-functional link. My first thought was that maybe it was a link but black for some reason.

Reply:
Done!